packages/core/src/module-resolver.ts

```diff
@@ -1016,7 +1016,7 @@ export class Resolver {
       if (pkg.packageJSON.exports) {
         let found = resolveExports(pkg.packageJSON, request.specifier, {
           browser: true,
-          conditions: ['default', 'imports'],
+          conditions: ['default', 'imports', 'browser'],
```

Comment:
todo: check if vite exports a list of conditions we can use (so we don't need to maintain this)

Reply:
TODO: order probably matters, do `import, module, browser, default`